sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentModelOperationSummary.cs

```diff
@@ -23,9 +23,9 @@ public partial class DocumentModelOperationSummary
         public DateTimeOffset LastUpdatedOn { get; }
 
         /// <summary>
-        /// A list of user-defined key-value tag attributes associated with the model.
+        /// URI of the resource targeted by this operation.
         /// </summary>
-        public IReadOnlyDictionary<string, string> Tags { get; }
+        public Uri ResourceLocation { get; }
```

Comment:
We don't need the `Tags` property in this file. We moved it to the generated part of this class.